.golangci.yaml

```diff
@@ -0,0 +1,3 @@
+run:
+  # Default timeout is 1m, up to give more room
+  timeout: 4m
```

Comment:
Can we also define a set of linters that we'd like to use? Are there any packages that need to be skipped (i.e. anything code generated)?

Reply:
I was poking around [linter options](https://golangci-lint.run/usage/linters/) and came up with this initial list locally:

```yaml
run:
  # Default timeout is 1m, up to give more room
  timeout: 4m

linters:
  enable:
  - asciicheck
  - deadcode
  - depguard
  - errorlint
  - gofmt
  - goimports
  - importas
  - prealloc
  - nestif
  - revive
  - misspell
  - stylecheck
  - tparallel
  - unconvert
  - unparam
  - whitespace
```

WDYT?

Reply:
So if I understand this correctly, these are additional linters that will run - right?

Reply:
In terms of things we could skip, the only generated code I can think of is:

`api/v1alpha1/zz_generate.deepcopy.go`